GetDKAN/dkan_starter
# Remove drush aliases from deploy

## Description
Removes drush aliases from deploy scripts.

## QA Tests
The ``.ahoy/site/.scripts/deploy.sh`` file uses `drush @alias` which doesn't work with our local docker setup and thus doesn't work with the new tests.  I've updated this so it should still work on Acquia.

- [x] `` "Installation detected, running deploy script"`` should happen for dkan_starter
- [ ] Should try and test on Acquia to make sure the deploy still works
- [x] Should work on probo
